.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Release checklist
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all the important changes in Changelog.md
+- [ ] Set the release date in Changelog.md
+- [ ] Set the current release version and data in Changelog.md
```

Comment:
```suggestion
- [ ] Set the current release version and data in README.md
```

Reply:
How can we set current release data?

Reply:
When we decide to release v0.2.0, we need to change the current version to v0.2.0 (2021-06-08) in README.md before making the release.

Reply:
So it's `Set the current release version in README.md`?

Reply:
Typo again. I meant "date".

Reply:
![image](https://user-images.githubusercontent.com/3974108/121201071-3eaf3a80-c842-11eb-95ca-0a82e15fd654.png)